src/icons/annotations/IconHighlightText.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import AccessibleSVG from '../accessible-svg';
+import { white } from '../../styles/variables';
+import { Icon } from '../iconTypes';
+
+const IconHighlightText = ({
+    className = '',
+    color = white,
+    height = 16,
```

Comment:
Seems like the other icons for annotations are a different size by default, should this be consistent?

Reply:
The other icons' sizes in this folder are not consistent, I saw 20/21/22/24